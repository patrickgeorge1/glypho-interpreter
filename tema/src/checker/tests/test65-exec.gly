888N###IHHH][[[7kkk8QQQ<eqjeO888F333