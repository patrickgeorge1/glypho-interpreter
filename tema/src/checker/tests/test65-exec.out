345345734
6758768
